docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -226,7 +226,6 @@ spec:
       nodePort: 30001
   type: NodePort
 EOF
-
```

Comment:
can we restore the newlines that were deleted? I think they were all in there to improve readability of commands in code blocks